libr/debug/p/debug_native.c

```diff
@@ -1259,6 +1259,52 @@ static int r_debug_desc_native_open (const char *path) {
 	return 0;
 }
 
+static int r_debug_setup_ownership (int fd, RDebug *dbg) {
```

Comment:
I would prefer RDebug don't touch the filesystem at all. and, as long as `r_debug_info` can be called from RCore you can just fchown in there.